docs/src/introduction.md

```diff
@@ -18,24 +18,24 @@ The language will be specified in detail in the rest of this book.
 
 ### CLI
 
-There is a command-line interface available for transpiling Air DSL files to Rust. There are also several example `.air` files written in the Air DSL which can be found in the `examples/` directory.
+There is a command-line interface available for transpiling AirScript files to Rust. There are also several example `.air` files written in the AirScript which can be found in the `examples/` directory.
 
 To use the CLI, first run:
 
 ```
 cargo build --release
 ```
 
-Then, run the `air-dsl` target with the `transpile` option and specify your input file with `-i`. For example:
+Then, run the `air-script` target with the `transpile` option and specify your input file with `-i`. For example:
 
 ```
-./target/release/air-dsl transpile -i examples/system.air
+./target/release/air-script transpile -i examples/system.air
```

Comment:
Thought for the future: I wonder if we should rename the executable to just `air`.

Reply:
I've moved this comment to #1 for discussion.